tests/bootstrap_common.php

```diff
@@ -51,3 +51,67 @@ function missing_ddtrace_class_fatal_autoloader($class)
 
 spl_autoload_register('\DDTrace\Tests\missing_ddtrace_class_fatal_autoloader', true);
 
+$phpunitVersionParts = class_exists('\PHPUnit\Runner\Version')
```

Comment:
The next three files related with bootstrap are changed because benchmarks were loading appsec Mock. This changes avoid that happening

Reply:
Basically I copied/pasted code from one file to another. All the code was already there but in different files